browser/default_protocol_handler_utils_win.cc

```diff
@@ -141,15 +141,11 @@ std::wstring FormatUserChoiceString(std::wstring_view ext,
       L"User Choice set via Windows User Experience "
       L"{D18B6DD5-6124-4341-9318-804003BAFA0B}";
 
-  const wchar_t* user_choice_fmt =
-      L"%ls%ls%ls"
-      L"%08lx"
-      L"%08lx"
-      L"%ls";
-
-  const std::wstring user_choice = base::StringPrintf(
-      user_choice_fmt, ext.data(), sid.data(), prog_id.data(),
-      file_time.dwHighDateTime, file_time.dwLowDateTime, user_experience);
+  const std::wstring file_time_str = base::ASCIIToWide(base::StringPrintf(
+      "%08lx%08lx", file_time.dwHighDateTime, file_time.dwLowDateTime));
+
+  const std::wstring user_choice = base::StrCat(
+      {ext.data(), sid.data(), prog_id.data(), file_time_str, user_experience});
```

Comment:
`.data()` calls here should not be necessary. `base::StrCat` should accept string_views as is.

Reply:
Thanks, fixed here f276b8175d61c800ed277c42c090a950359a5d54.